consultation_analyser/factories.py

```diff
@@ -119,12 +113,10 @@ class ThemeFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = models.Theme
 
-    label = factory.LazyAttribute(lambda _o: generate_dummy_topic_label())
+    keywords = factory.LazyAttribute(lambda _o: generate_dummy_topic_keywords())
+    short_description = factory.LazyAttribute(lambda _o: faker.sentence())
     summary = factory.LazyAttribute(lambda _o: faker.sentence())
```

Comment:
Do we need an is_outlier value here? (Probably false by default)

Reply:
Good point - have made it false by default.